community/kernel/src/main/java/org/neo4j/helpers/Format.java

```diff
@@ -106,14 +108,24 @@ public static String time( Date date, TimeZone timeZone )
 
     public static String bytes( long bytes )
     {
-        double size = bytes;
-        for ( String suffix : BYTE_SIZES )
+        return suffixCount( bytes, BYTE_SIZES, KB );
+    }
+
+    public static String count( long count )
+    {
+        return suffixCount( count, COUNT_SIZES, 1_000 );
```

Comment:
do not think its an idea that someone will ever see TB as count suffix

Reply:
There will be some day. Although I see no point in removing it to save a few characters in the code, do you?

Reply:
'TB' is not a valid count size prefix right? (and inside suffixCount you can see that it will be used for case when number is big)

Reply:
`k`, `M`, `G`, `T`... so you mean `T` don't you? And why is that not OK?

Reply:
no no, 'T' is fine, but if you will go into `suffixCount` method and if `value` is big enough  you will unconditionally use `TB` suffix